pkg/tfgen/generate_python.go

```diff
@@ -666,6 +684,96 @@ func (g *pythonGenerator) emitPackageMetadata(pack *pkg) error {
 	return nil
 }
 
+// Emits property conversion tables for all properties recorded using `recordProperty`. The two tables emitted here are
+// used to convert to and from snake case and camel case.
+func (g *pythonGenerator) emitPropertyConversionTables(tableModule *module) error {
+	w, err := g.openWriter(tableModule, "tables.py", false)
+	if err != nil {
+		return err
+	}
+
+	defer contract.IgnoreClose(w)
+	var allKeys []string
+	for key := range g.snakeCaseToCamelCase {
+		allKeys = append(allKeys, key)
+	}
+
+	sort.Strings(allKeys)
```

Comment:
Tiny nit: can you nudge this line up against the preceding loop? I very nearly missed it.